apis/v1beta1/gatewayclass_types.go

```diff
@@ -186,6 +187,49 @@ const (
 	GatewayClassReasonWaiting GatewayClassConditionReason = "Waiting"
 )
 
+const (
+	// This condition indicates whether the GatewayClass supports the version(s)
+	// of Gateway API CRDs present in the cluster. This condition MUST be set by
```

Comment:
Since this condition "MUST be set", would it make sense to update the conformance tests too?

Reply:
Yes, I think we'll want to have corresponding updates to the conformance tests. I'd initially thought those should be a separate PR, but they may be simple enough to combine here.